src/resdk/tables/base.py

```diff
@@ -213,6 +223,30 @@ def _metadata_version(self) -> str:
         )
         return version
 
+    @property
+    @lru_cache()
+    def _qc_version(self) -> str:
+        """Return server QC data version.
+
+        The versioning of QC Data on the server is determined by the hash of
+        the tuple of sorted QC data objects ids.
+
+        :return: data version
+        """
+        mqc_ids = []
+        for mqc in self.collection.data.filter(
```

Comment:
Would not the same be achieved by 

``mqc_ids = self.collection.data.filter(type="data:multiqc", status="OK", fields=["id", "entity__id"], entity__isnull=True)``  ?

Reply:
Good point, thanks.